pkgs/applications/networking/irc/catgirl/default.nix

```
@@ -0,0 +1,21 @@
+{lib, stdenv, fetchurl, ncurses, libressl, man, pkg-config, ctags}:
+
+stdenv.mkDerivation rec {
+  name = "catgirl-1.6";
```

```suggestion
  pname = "catgirl";
  version = "1.6";
```

see https://nixos.org/manual/nixpkgs/unstable/#sec-using-stdenv ([RFC 0035](https://github.com/NixOS/rfcs/pull/35))